tests/helpers.py

```diff
@@ -0,0 +1,96 @@
+import pickle
```

Comment:
Considering combining this file with `tests/fixtures.py`

Reply:
I didn't read the file, but basically helpers are small helper functions and fixtures are larger constructs (generally they subclass unittest.TestCase, but not always).